include/terarangerone/terarangerone.h

```diff
@@ -102,6 +102,8 @@ class TerarangerOne
   SerialPort * serial_port_;
   boost::function<void(uint8_t)> serial_data_callback_function_;
 
+  sensor_msgs::Range range_msg_;
```

Comment:
make range message member to avoid instantiating each time in serial port callback